tests/helpers.py

```diff
@@ -0,0 +1,113 @@
+"""Module to help to create tests."""
```

Comment:
Isn't it time to move this helper lib to kytos core? At least the part that is being duplicated in every NApp.

Reply:
I agree with @hdiogenes, we should have these in a central place.